### PR TITLE
Tidy cfgspec/parsec slightly

### PR DESCRIPTION
**Pull request by MetRonnie**
Update: I realised the original aim of this PR wasn't practical. Instead I've just tidied the code & comments ever so slightly.

This is a small change with no associated Issue.

**Requirements check-list**
- [X] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [X] Contains logically grouped changes (else tidy your branch by rebase).
- [X] Does not contain off-topic changes (use other PRs for other changes).
- [X] Does not need tests (no real changes).
- [X] No change log entry required.
- [X] No documentation update required.
- [X] No dependency changes.

### CODE REVIEW

**Comment by MetRonnie** [comment]
Something isn't right, I'm looking into it

**Comment by oliver-sanders** [comment]
`None` is basically the default anyway, technically the config node defaults to unset but the end result is the same no?

**Comment by MetRonnie** [comment]
There is still the issue of: if you have `UTC mode = True` in your global config and you want to unset it to the default in the flow config, well, you can't exactly unset it. However, as Oliver has pointed out, just setting `UTC mode = False` in flow config is probably good enough? (UTC mode is the only bool setting in both flow & global cfgspec)

**Review by wxtim — APPROVED** [review]
Happy with this.

File: cylc/flow/parsec/OrderedDict.py [hunk 1]
```diff
@@ -95,12 +93,9 @@ def __bool__(self):
...
     def prepend(self, key, value):
-        """Prepend new item in the ordered dict.
-
-        https://stackoverflow.com/questions/16664874/
-           how-can-i-add-an-element-at-the-top-of-an-ordereddict-in-python
-
-        """
+        """Prepend new item in the ordered dict."""
+        # https://stackoverflow.com/questions/16664874/
+        #   how-can-i-add-an-element-at-the-top-of-an-ordereddict-in-python
```
Comment (oliver-sanders) [inline cylc/flow/parsec/OrderedDict.py:98]: I'm confused why are you converting docstrings to comments?

What's the gain.

Comment (MetRonnie) [reply cylc/flow/parsec/OrderedDict.py:98]: Not much gain, but I figure those don't describe what the methods do, so they're not suitable to be docstrings

File: cylc/flow/cfgspec/globalcfg.py [hunk 2]
```diff
@@ -134,7 +134,6 @@
                 Conf('interval', VDR.V_INTERVAL, desc='''
                     The interval with which this plugin is run.
                 ''')
-                Conf('foo', VDR.V_STRING, default='X')
```
Comment (hjoliver) [inline cylc/flow/cfgspec/globalcfg.py:137]: Huh? :thinking:

**Review by hjoliver — APPROVED** [review]
Looks fine to me; Oliver can merge if happy with your docstring/comment response.

File: cylc/flow/cfgspec/globalcfg.py (hunk 2 above)
Comment (oliver-sanders) [reply cylc/flow/cfgspec/globalcfg.py:137]: Yeah, not sure how that one snuck in.

**Pull request merged**